grammars.yml

```diff
@@ -1065,6 +1065,7 @@ vendor/grammars/vscode-gleam:
 vendor/grammars/vscode-go:
 - go.mod
 - go.sum
+- go.work
```

Comment:
🤔 where did this come from? I don't see it in the grammar. This file is kept up to date based on the content of the grammars and shouldn't be manually modified.

Reply:
Removed the non-existing grammar here